tests/unit/test_viz.py

```diff
@@ -14,10 +14,20 @@
 """"Unit tests for functions in the viz module."""
 
 import math
+from datetime import datetime
+
 import numpy as np
 from numpy.random import default_rng as rng
+import pytest
+import matplotlib.pyplot as plt
+
 
-from flamingpy.utils.viz import to_pi_string
+from flamingpy.utils.viz import to_pi_string, draw_EGraph
+from flamingpy.codes.graphs import EGraph
+from flamingpy.codes import SurfaceCode
+
+now = datetime.now()
+int_time = int(str(now.year) + str(now.month) + str(now.day) + str(now.hour) + str(now.minute))
```

Comment:
I don't think this is necessary here -- we do not need randomness in the tests below.